benchmarks/src/bin/parquet_filter_pushdown.rs

```diff
@@ -109,6 +111,13 @@ async fn main() -> Result<()> {
     Ok(())
 }
 
+#[derive(Debug, Clone)]
+struct ParquetScanOptions {
```

Comment:
this was replicated for the benchmark code as I felt such a struct was the easiest to understand for this matrix strategy